src/main/java/pl/piekoszek/gorskimatches/equation/QuizzesGenerator.java

```diff
@@ -17,6 +17,12 @@ public QuizzesGenerator(SolutionToQuizzesMapper solutionToQuizzesMapper, Equatio
                 solutionsByQuiz.get(quiz).addAll(solution);
             });
         }
+        for (String correctEquation : mathematicallyCorrectEquations.mathematicallyCorrectEquations()) {
```

Comment:
object should be called `equationGenerator`, not `mathematicallyCorrectEquations` didn't catch it before